testsuite/openshift/client.py

```diff
@@ -213,6 +213,18 @@ def select_operator(apiobject):
 
         return self.select_resource("pods", narrow_function=select_operator)
 
+    def get_apicast_operator(self):
```

Comment:
it would be nice if this was property for now

Reply:
fixed.